tests/test_adaptor.py

```diff
@@ -174,6 +174,53 @@ def test_write_works(self):
 
         pin.write.assert_called_with(100)
 
+    def test_read_sets_up_pin(self):
+        self.edison.digital_read(5)
+
+        self.assertTrue(
```

Comment:
You should be able to use `assertIn` here, which will give us a slightly better output if this starts failing.